clang-tools-extra/docs/clang-tidy/checks/bugprone/implicit-widening-of-multiplication-result.rst

```diff
@@ -50,7 +50,7 @@ Options
    If the multiplication operands are compile-time constants (like literals or
    are ``constexpr``) and fit within the source expression type, do not emit a
    diagnostic or suggested fix.  Only considers expressions where the source
-   expression is a signed integer type.
+   expression is a signed integer type.  Defaults to ``false``.
```

Comment:
single `

Reply:
The other options all use double ticks for the default value; I'd think it should be the same for this one

Reply:
those options are invalid, but that can be fixed in general for all options at once anyway